metron-platform/metron-writer/src/test/java/org/apache/metron/writer/BulkWriterComponentTest.java

```diff
@@ -118,6 +118,20 @@ public void writeShouldProperlyAckTuplesInBatch() throws Exception {
   }
 
   @Test
+  public void writeShouldProperlyAckTuplesWhenWriterDisabled() throws Exception {
```

Comment:
I combined the test for a disabled sensor into the previous test for enabled sensors.  Is this ok?  Otherwise we'll have 2 tests that look almost identical.